config/environments/test.rb

```diff
@@ -90,6 +90,16 @@
       }
     })
 
+  # This config is has the same email as one of the fixture users. When it is used to login, instead of signing up a new user
+  # it should link this oauth config to the fixture user.
+  OmniAuth.config.mock_auth[:github4] = OmniAuth::AuthHash.new({
```

Comment:
Can you kindly use existing fixtures of possible?
If it is mandatory to create fixtures then create for all the four providers.
Thanks

Reply:
No, I need to make a new one here because the fixture has to have the same email as one of the normal login fixture users. I will add the config for the four providers.